subprojects/jacoco/src/main/java/org/gradle/testing/jacoco/tasks/JacocoReportBase.java

```diff
@@ -48,6 +51,7 @@
 @Incubating
 public abstract class JacocoReportBase extends JacocoBase {
 
+    private Set<JacocoTaskExtension> taskExtensions = new HashSet<JacocoTaskExtension>();
```

Comment:
The report task can have includes/excludes according to the documentation: http://www.eclemma.org/jacoco/trunk/doc/ant.html.
JacocoReportBase can be associated with more than one task extension, and combining all their includes/excludes will probably be correct in most cases, but exceptions are possible.
Maybe transfer could be the default behavior, but overridable.